web/themes/custom/votegov/templates/component/info-card.html.twig

```diff
@@ -0,0 +1,19 @@
+<div class="vote-info-card">
+	<div class="vote-info-card__container">
```

Comment:
probably can remove container

Reply:
removed container div

Reply:
I kept the container since each section of the card had a padding of 32 px all around